pkg/colorful-hebrew.man

```diff
@@ -0,0 +1,31 @@
+.\" דף הוראות של Colorful
+.\" צרו קשר עם veg@svgames.pl לתיקוני בעיות או שגיאות.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
```

Comment:
"Games Manual" should be translated. The date should also reflect the manpage modification date.